ansible_ai_connect/ai/tests/test_apps.py

```diff
@@ -84,86 +84,6 @@ def test_mock_client(self):
             app_config.get_model_pipeline(ModelPipelineCompletions), DummyCompletionsPipeline
         )
 
-    @override_settings(ENABLE_ARI_POSTPROCESS=True)
-    @override_settings(ANSIBLE_AI_MODEL_MESH_CONFIG=mock_config("dummy"))
```

Comment:
You should still keep the tests where `ENABLE_ARI_POSTPROCESS=False`

Reply:
done